packages/titanium-single-select/src/titanium-single-select.ts

```diff
@@ -320,14 +321,21 @@ export class TitaniumSingleSelectElement extends LitElement {
       font-weight: 400;
     }
 
-    search-suggestions hr {
+    hr {
+      position: absolute;
```

Comment:
absolute position this so that it wouldn't take up any actual height as it is for visual effect only.